django-stubs/forms/fields.pyi

```diff
@@ -378,6 +378,10 @@ class SlugField(CharField):
     def __init__(
         self,
         allow_unicode: bool = ...,
+        max_length: Optional[Any] = ...,
```

Comment:
`Any` or `int`?

Reply:
It accepts any value and tries to cast to `int`. So, weird things like `max_length=Decimal("10.3")` are acceptable.

Also, the stub for `CharField` defines it as `Any` and inherited classes cannot use a more specific type.